Automattic/woocommerce-payments
# Refactor upe checkout into its own file

Fixes #2163

#### Changes proposed in this Pull Request
Separate UPE classic checkout into its own file. This will reduce the size of classic checkout and make both scripts easier to manage.









#### Testing instructions

* Test you can checkout using UPE and a regular card (4242424242424242 )
* Test you can checkout using UPE and a card that requires authorization ( 4000002500003155 )
* Test you can checkout using UPE and save the payment method
* Test you can checkout using a saved Payment Method
* Test you can save a payment method from My Account > Manage Payment Methods > Add Payment Method

Behavior when UPE is not enabled should work as before, this is covered by existing E2E tests.





-------------------

- [ ] Covered with tests (or have a good reason not to test in description ☝️) - E2E tests will be part of #1903
